pkg/ebpf/gadgets/http/tracer/bpf/http-sniffer.c

```diff
@@ -0,0 +1,623 @@
+#include "sniffer_strcuts.h"
+#include <bpf/bpf_helpers.h>
+#include <bpf/bpf_tracing.h>
+#include <bpf/bpf_core_read.h>
+
+typedef unsigned short int sa_family_t;
+
+struct
+{
+    __uint(type, BPF_MAP_TYPE_PERF_EVENT_ARRAY);
+} events SEC(".maps");
+
+struct
+{
+    __uint(type, BPF_MAP_TYPE_HASH);
+    __uint(max_entries, 4096);
+    __type(key, __u64);
+    __type(value, struct pre_accept_args);
+} pre_accept_args_map SEC(".maps");
```

Comment:
Add a comment on top of each map what it is used for.

Reply:
DONE